src/main/java/com/ss/mqtt/broker/util/DebugUtils.java

```diff
@@ -0,0 +1,91 @@
+package com.ss.mqtt.broker.util;
+
+import com.google.gson.ExclusionStrategy;
+import com.google.gson.FieldAttributes;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.ss.rlib.common.util.ReflectionUtils;
+import com.ss.rlib.common.util.array.Array;
+import com.ss.rlib.common.util.array.ArrayFactory;
+import org.jetbrains.annotations.NotNull;
+
+import java.lang.StackWalker.Option;
+import java.lang.reflect.Field;
+import java.util.Set;
+
+public class DebugUtils {
```

Comment:
Add validation for enabled debug mode for all methods

Reply:
It's configurable on logger level, not an 'application property'